legacy/package_libpng_1_6_7/tool_dependencies.xml

```diff
@@ -1,20 +1,28 @@
 <?xml version="1.0"?>
-<tool_dependency>
+<tool_dependency> 
+  <package name="zlib" version="1.2.8">
+    <repository name="package_zlib_1_2_8" owner="iuc" prior_installation_required="True" />
+  </package>
   <package name="libpng" version="1.6.7">
-      <install version="1.0">
-          <actions>
-              <action type="download_by_url">http://downloads.sourceforge.net/project/libpng/libpng16/older-releases/1.6.7/libpng-1.6.7.tar.gz</action>
-                <action type="autoconf"/>
-                <action type="set_environment">
-                  <environment_variable action="prepend_to" name="PATH">$INSTALL_DIR/bin</environment_variable>
-                  <environment_variable action="set_to" name="LIBPNG_ROOT">$INSTALL_DIR</environment_variable>
-                  <environment_variable action="set_to" name="LIBPNG_LIB_PATH">$INSTALL_DIR/lib</environment_variable>
-                  <environment_variable action="set_to" name="LIBPNG_INCLUDE_PATH">$INSTALL_DIR/include</environment_variable>
-                  <environment_variable action="prepend_to" name="LD_LIBRARY_PATH">$INSTALL_DIR/lib</environment_variable>
-                  <environment_variable action="prepend_to" name="PKG_CONFIG_PATH">$INSTALL_DIR/lib/pkgconfig</environment_variable>
-                </action>
-            </actions>
-        </install>
-        <readme />
-    </package>
+    <install version="1.0">
+      <actions>
+        <action type="download_by_url">http://downloads.sourceforge.net/project/libpng/libpng16/older-releases/1.6.7/libpng-1.6.7.tar.gz</action>
+        <action type="set_environment_for_install">
+          <repository name="package_zlib_1_2_8" owner="iuc" prior_installation_required="True">
+            <package name="zlib" version="1.2.8" />
+          </repository>
+        </action>
+        <action type="autoconf">CPPFLAGS="-I$ZLIB_ROOT_PATH/include" LDFLAGS="-L$ZLIB_ROOT_PATH/lib"</action>
```

Comment:
This shouldn't be necessary, as we set LD_LIBRARY_PATH and C_INCLUDE_PATH/CPLUS_INCLUDE_PATH in zlib. Also I don't think this can have any effect, since the line in galaxy is
```
pre_cmd = './configure --prefix=$INSTALL_DIR %s && make && make install' % configure_opts
```
```
(.venv)mariuss-MBP:~ marius$ echo "test" BLA=blabla
test BLA=blabla
(.venv)mariuss-MBP:~ marius$ echo $BLA

```